web/src/helpers/hooks/useIdeState.js

```diff
@@ -14,19 +22,37 @@ module stick(basewid, angl){
         sphere(7);
         translate([0,0,10])sphere(9);
     }
-}`
+}`,
+  cadQuery: `import cadquery as cq
+from cadquery import exporters
+
+diam = 5.0
+
+result = (cq.Workplane().circle(diam).extrude(20.0)
+          .faces(">Z").workplane(invert=True).circle(1.05).cutBlind(8.0)
+          .faces("<Z").workplane(invert=True).circle(0.8).cutBlind(12.0)
+          .edges("%CIRCLE").chamfer(0.15))
+
+# exporters.export(coupler, "/home/jwright/Downloads/coupler.stl", exporters.ExportTypes.STL)
+
+show_object(result)
+`,
+}
 
 export const codeStorageKey = 'Last-openscad-code'
```

Comment:
Tiny thing for future, this key might want to be package-agnostic at some point.

Reply:
Yup fixed this up.